src/utils/store.js

```diff
@@ -4,8 +4,15 @@ const store = new Store()
 
 if (store.get('version') !== 4) {
   store.clear()
-  store.set('seenWelcome', false)
-  store.set('config', null)
+
+  // default config
+  store.set('config', {
```

Comment:
```suggestion
  store.set('ipfsConfig', {
```

`config` is a really generic name for a property in a config store!